src/licensedcode/plugin_license.py

```diff
@@ -167,7 +176,7 @@ def setup(self, **kwargs):
         loaded index.
         """
         from licensedcode.cache import populate_cache
-        populate_cache()
+        populate_cache(additional_directories=kwargs.get('additional_directories'))
```

Comment:
I think its cleaner to have the `kwargs.get()` call set as a variable first
```suggestion
        additional_directories = kwargs.get('additional_directories')
        populate_cache(additional_directories=additional_directories)
```